### DISCUSSION TITLE
Bill with different taxes

### DISCUSSION THREAD

**Discussion by chrisschx** [stephanstapel/ZUGFeRD-csharp#281] [General]
Hi,
if a bill has different taxes, you can add the taxes for each position and in the bill itself.

To add the different taxes in the bill, is it enough to call AddApplicableTradeTax for each tax?

Best regards,
Christian

**Comment by stephanstapel** [comment]
Yes, that should be the case. See the example on the project page: https://github.com/stephanstapel/ZUGFeRD-csharp
